src/app/services/items/index.ts

```diff
@@ -1,9 +1,30 @@
+import { Observable, Subject } from 'rxjs';
+import { ItemsList } from '@core/pages/main/namespace';
 import ApiService from '../config';
 
+// for Saga usage
 export const getItemsService = (payload: { id: number }) => {
   return ApiService.get('/items', {
     id: payload.id,
   }).then((data: any) => {
     return data.data;
   });
 };
+
+// for Epic usage
+class ItemsService {
+  public getItems(payload: { id: number }): Observable<ItemsList> {
```

Comment:
do we really need a new class instance in this case? can't it be exported as a simple function, like in above?